docs/vector-indexes.md

```diff
@@ -72,11 +72,59 @@ from wagtail_vector_index.index.models import EmbeddableFieldsVectorIndex
 
 class MyEmbeddableFieldsVectorIndex(EmbeddableFieldsVectorIndex):
     querysets = [
-        MyModel.objects.all(),
-        MyOtherModel.objects.filter(name__startswith="AI: "),
+        InformationPage.objects.all(),
+        BlogPage.objects.filter(name__startswith="AI: "),
     ]
 ```
 
+We're hoping to handle this automatically in the near future. But for now, you'll also need to override a couple of methods on your class to allow it to succesfully retrieve content for indexing. For example:
+
+```python
+# indexes.py
+from wagtail.models import Page
+
+
+class MyEmbeddableFieldsVectorIndex(EmbeddableFieldsVectorIndex):
+    ...
+
+    def get_converter(self, model_class=None):
+        return self.get_converter_class()(model_class or Page)
+
+    def get_documents(self):
+        all_documents = []
+
+        for queryset in self._get_querysets():
+            converter = self.get_converter(model_class=queryset.model)
+            documents = list(
+                converter.bulk_to_documents(
+                    queryset.prefetch_related("embeddings"),
+                    embedding_backend=self.embedding_backend,
+                )
+            )
+            all_documents.extend(documents)
+
+        return all_documents
+```
+
+Custom indexes need to be 'registered' with the `wagtail-vector-index` before they can be used. The best place to do this is in the `ready()` method of an `AppConfig` class within your project. You may find it helpful to save your custom index and any other related code to a new `vector_index` app in your project; in which case, `vector_index/apps.py` might look something like this:
```

Comment:
Perhaps this section on registering indexes should be under the "Creating your own Indexes" heading (above the Indexing Multiple Models h3)?

Reply:
Okay, cool. I can then reference that from here.

Reply:
On second thoughts, I don't think it's necessary there, because it happens automatically when a page model subclasses `GeneratedIndexMixin`

Reply:
Great OK makes sense to me!